webcompat/static/js/lib/bugform.js

```diff
@@ -77,9 +77,14 @@ function BugForm() {
     );
     this.problemType.on("change", _.bind(this.checkProblemTypeValidity, this));
     this.uploadField.on("change", _.bind(this.checkImageTypeValidity, this));
-    this.osField
-      .add(this.browserField)
-      .on("blur input", _.bind(this.checkOptionalNonEmpty, this));
+    this.osField.on(
+      "blur",
```

Comment:
Is removing the `input` handler intentional?

Reply:
adding both would mean to add a bind to both. I don't feel this is needed. Am I missing something?

Reply:
Not sure I follow, let me try again:

Before we had `$(foo).on("blur input", bar)`. Now we have $(foo).on("blur", bar)`, meaning we're not handling the `input` event.

What do you mean by adding `bind` to both? I don't follow. Thanks!

Reply:
Sorry for the short and confusing answer. Okay, let me rephrase it. :)

So, we add `blur` and `input` as events to trigger the function. I think, `blur` is enough as this will add / remove the checkmark as needed and we don't fire the function on every change of the input value. So the answer to your question `Is removing the input handler intentional?` is yes. 

If you think, this feels weird or does not cover all use cases, I am ok to put it back. I guess, this is mostly about UX preference.

Reply:
SGTM. Let's merge now and if it feels wrong later on, file a bug to revisit/change it back.